src/main/kotlin/no/nav/syfo/aktivitetskrav/database/AktivitetskravVarselRepository.kt

```diff
@@ -108,7 +115,47 @@ private fun Connection.createAktivitetskravVarselPdf(varselId: Int, pdf: ByteArr
     return varselPdfs.first()
 }
 
-private fun ResultSet.toPAktivitetskravVarsel(): PAktivitetskravVarsel =
+const val queryGetIkkeJournalforteVarsler = """
+    SELECT av.*, avp.pdf as pdf, a.personident as personident 
+    FROM aktivitetskrav_varsel av 
+    INNER JOIN aktivitetskrav_varsel_pdf avp
+    ON av.id = avp.aktivitetskrav_varsel_id
+    INNER JOIN aktivitetskrav_vurdering avu
+    ON av.aktivitetskrav_vurdering_id = avu.id
+    INNER JOIN aktivitetskrav a
+    ON avu.aktivitetskrav_id = a.id
+    WHERE av.journalpost_id IS NULL
+"""
+
+fun DatabaseInterface.getIkkeJournalforteVarsler(): List<Triple<PersonIdent, PAktivitetskravVarsel, ByteArray>> {
+    return this.connection.use { connection ->
+        connection.prepareStatement(queryGetIkkeJournalforteVarsler).use {
+            it.executeQuery().toList { Triple(PersonIdent(getString("personident")), toPAktivitetskravVarsel(), getBytes("pdf")) }
+        }
+    }
+}
+
+const val queryUpdateJournalpostId = """
+    UPDATE aktivitetskrav_varsel
+    SET journalpost_id = ?
+    WHERE uuid = ?
+"""
+
+fun DatabaseInterface.updateVarselJournalpostId(varsel: AktivitetskravVarsel, journalpostId: String) {
+    this.connection.use { connection ->
+        connection.prepareStatement(queryUpdateJournalpostId).use {
+            it.setString(1, journalpostId)
+            it.setString(2, varsel.uuid.toString())
+            val updated = it.executeUpdate()
+            if (updated != 1) {
+                throw SQLException("Expected a single row to be updated, got update count $updated")
+            }
+        }
+        connection.commit()
+    }
+}
+
+fun ResultSet.toPAktivitetskravVarsel(): PAktivitetskravVarsel =
```

Comment:
Måtte gjøre denne public for å kunne bruke den i TestDatabase (som burde være greit). Kanskje vi skulle samlet disse ResultSet-mapper-funksjonene i en egen fil.

Reply:
Jeg tenker at de fint kan være i samme fil som konteksten de blir brukt 😁 Hvis man lager en egen fil for å samle extension functions så er jeg redd for at vi kan miste litt grunnen til hvorfor de ble laget i utgangspunktet.

Reply:
Enig i det, men siden den brukes i kontekst av en query tenker jeg den også bør være tilgjengelig for `TestDatabase` og queryen(e) der.  Har ikke så lyst til å duplisere dette `TestDatabase` og måtte oppdatere flere steder hver gang varsel-tabellen endrer seg. 
Gjør samtidig de andre funksjonene og queryene her private slik at de ikke kan brukes utenfor repository.